Subject: manually set python shebang
Description:
 Since this isn't really a standard python package, there is not even a
 setup.py, manually set the shebang to the Debian standard.

Origin: vendor
From: Hans-Christoph Steiner <hans@eds.org>
Reviewed-by: Marcos Marado <mindboosternoori@gmail.com>
Last-Update: 2020-02-24
Applied-Upstream: not-needed

--- a/repo
+++ b/repo
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 # -*- coding:utf-8 -*-
 #
 # Copyright (C) 2008 The Android Open Source Project
